src/components/auth/SettingsModal.tsx

```diff
@@ -45,11 +46,15 @@ export const SettingsMenuItem = React.memo(function SettingsMenuItem({ finalRef
 
     const profile = useSelector((state: AppState) => ProfileSelectors.getProfile(state, userId ?? ""));
     const toskiPlayer = getPlayerName(profile ? profile.id : "");
+    const defaultMoxfieldLogo = "https://pbs.twimg.com/profile_images/1674989472839094273/p7a37K9W_400x400.jpg";
+    const errorMoxfieldLogo = "https://upload.wikimedia.org/wikipedia/commons/4/4e/OOjs_UI_icon_error-destructive.svg";
 
     const favoriteCommanderId = profile && profile.favoriteCommanderId ? profile.favoriteCommanderId : "no value";
     const [commanderSelectValue, setCommanderSelectValue] = useState<string>(() => favoriteCommanderId);
     const [isRememberMe, setIsRememberMe] = useState<boolean>(accessTokenFromState !== null);
     const [showMoxfieldLinker, setShowMoxfieldLinker] = useState<boolean>(false);
+    const [moxfieldInputValue, setMoxfieldInputValue] = useState<string>("");
```

Comment:
super nit: unclear what "moxfieldInputValue" is representing just by looking at this. maybe "moxfieldIdInputValue"?